src/System.Private.Xml.Linq/tests/TreeManipulation/SaveWithFileName.cs

```diff
@@ -0,0 +1,223 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.XmlDiff;
+using CoreXml.Test.XLinq;
+using Microsoft.Test.ModuleCore;
+using XmlCoreTest.Common;
+using Xunit;
+
+namespace XLinqTests
```

Comment:
>namespace [](start = 0, length = 9)

where did you get these ugly tests :) there are existing tests here, perhaps you could rip those off? 
[http://index/?query=XStreamingElement&rightProject=XLinq.UnitTest&file=UnitTest.cs&line=3407](http://index/?query=XStreamingElement&rightProject=XLinq.UnitTest&file=UnitTest.cs&line=3407)

they include the various Save methods